### ISSUE TITLE
Support x-nullable vendor extension

### ISSUE THREAD

**Issue by budde377** [ferdikoomen/openapi-typescript-codegen#380]
**Describe the solution you'd like**
Since `nullable` isn't supported in OpenApi v2, some frameworks (e.g., [drf-yasg](https://drf-yasg.readthedocs.io/en/stable/)) are using the `x-nullable` vendor extension to indicate if e.g. property is nullable. I believe we should use this to mark a property as nullable:

The specification

```json
{
    "ModelWithNullableString": {
        "required": ["requiredProp"],
        "description": "This is a model with one string property",
        "type": "object",
        "properties": {
            "prop": {
                "description": "This is a simple string property",
                "type": "string",
                "x-nullable": true
            },
            "requiredProp": {
                "description": "This is a simple string property",
                "type": "string",
                "x-nullable": true,
            }
        }
    }
}
```

should yield

```typescript
enum ModelWithNullableString {
    prop?: string | null,
    requiredProp: string | null
}
```

**Comment by ferdikoomen** [comment]
@budde377 I merged the PR, 0.5.1 is available in NPM

**Comment by budde377** [comment]
Thank you for your quick response and great project! 🙏
